src/hotspot/share/gc/g1/g1FullGCPrepareTask.hpp

```diff
@@ -57,17 +57,19 @@ class G1FullGCPrepareTask : public G1FullGCTask {
     G1CMBitMap* _bitmap;
     G1FullGCCompactionPoint* _cp;
     uint _humongous_regions_removed;
+    uint _open_archive_regions_freed;
```

Comment:
Since we no longer use these counters to update the sets, it is enough to just have one counter to track if any region has been freed. Or use a boolean if you prefer that.

Reply:
Fixed.